meinberlin/apps/ideas/models.py

```diff
@@ -104,6 +107,50 @@ def remark(self):
             item_object_id=self.id
         ).first()
 
+    @property
```

Comment:
would it make sense to expose this to the api, so we can remove that logic on frontend? is that even possible?

Reply:
good idea, but maybe in another PR, so that this can be tested?